src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -771,6 +748,246 @@ const schema = {
         }
       }
     },
+    form8940: {
+      type: 'object',
+      properties: {
+        unemployability: {
+          type: 'object',
+          properties: {
+            hospitalizationHistory: {
+              type: 'array',
+              items: {
+                type: 'object',
+                properties: {
+                  hospitalizationDateRange: {
+                    type: "string"
+                  },
+                  hospitalName: {
+                    type: 'string',
+                    minLength: 1,
+                    maxLength: 100
+                  },
+                  hospitalAddress: {
+                    type: 'object',
+                    properties: _.omit(
+                      ['addressLine3', 'country'],
+                      baseAddressDef.properties
+                    )
+                  }
+                }
```

Comment:
Do you mind moving hospitalizationDateRange after hospitalAddress so it shows up in the proper order on the page?

Reply:
`hospitalizationHistory` is going away switching to using `hospitalProvidedCare`

Reply:
okay. can you move the daterange so that it shows up as the last field on the page?